### PR TITLE
CE#34700: The composer dependencies have been updated

### PR DESCRIPTION
**Pull request by andrewbess**
### Description (*)

This pull request provides the updates for composer dependencies described below
- allure-framework/allure-phpunit
- allure-framework/allure-php-api
- jms/metadata
- phpstan/phpstan
- webonyx/graphql-php
- magento/zendframework1 dev-master as 1.14.6

### Related Pull Requests
https://github.com/magento/partners-magento2ee/pull/661

### Fixed Issues (if relevant)

1. Partially fixes for magento/magento2#34700

### Contribution checklist (*)
 - [ ] Pull request has a meaningful description of its purpose
 - [ ] All commits are accompanied by meaningful commit messages
 - [ ] All new or changed code is covered with unit/integration tests (if applicable)
 - [ ] All automated tests passed successfully (all builds are green)

### CODE REVIEW

**Comment by andrewbess** [comment]
@magento run all tests

**Review by ihor-sviziev — APPROVED** [review]

**Comment by magento-engcom-team** [comment]
Hi @ihor-sviziev, thank you for the review.
[ENGCOM-9357](https://jira.corp.magento.com/browse/ENGCOM-9357) has been created to process this Pull Request
:eight_spoked_asterisk: @ihor-sviziev, could you please add one of the following labels to the Pull Request?

| Label | Description |
|--------------------------------|-----------------------------------------------------------------|
| ```Auto-Tests: Covered``` | All changes in Pull Request is covered by auto-tests |
| ```Auto-Tests: Not Covered``` | Changes in Pull Request requires coverage by auto-tests |
| ```Auto-Tests: Not Required``` | Changes in Pull Request does not require coverage by auto-tests |

**Comment by andrewbess** [comment]
@magento run all tests

File: composer.json [hunk 1]
```diff
@@ -381,5 +381,12 @@
...
     },
-    "prefer-stable": true
+    "prefer-stable": true,
+    "repositories": [
+        {
+            "type": "vcs",
+            "url": "https://github.com/magento/zf1.git"
+        }
+    ],
+    "minimum-stability": "dev"
```
Comment (fascinosum) [inline composer.json:391]: please delete this extra declarations

**Comment by xmav** [comment]
@magento import code to magento-trigger/magento2ce

**Comment by m2-github-services** [comment]
@xmav the branch with the code is successfully imported into the `magento-trigger/magento2ce` repository. Branch name: `imported-andrewbess-magento2ce-34737`.

**Pull request merged**